### ISSUE TITLE
[Task] Winner name mismatch REGISTRANTS Over WINNERS 

### ISSUE THREAD

**Issue by drasticdpk** [topcoder-platform/work-manager#1009]
### Expected behavior
Registrants  and Winner name should be same for Task.
 

### Screenshot/screencast
 
![image](https://user-images.githubusercontent.com/25297859/102599640-dbbbd100-4145-11eb-95c6-9c971c97f024.png)

 https://www.topcoder.com/challenges/849e4d4f-0c3f-44a9-9ac2-318db739829c?tab=registrants

**Comment by acshields** [comment]
@maxceem - has this one come across your plate yet, with Vikas out? If not, can you take a look?

**Comment by maxceem** [comment]
@acshields I'm on it now.

@drasticdpk I cannot reproduce this issue. I've processed a couple of payment on Prod it works correctly for me. Maybe you did some specific steps, or had some issue on the way?

@vikasrohit do you have any idea on why this might happen. I see some tasks are having this issue:
- https://www.topcoder.com/challenges/849e4d4f-0c3f-44a9-9ac2-318db739829c?tab=winners
- https://www.topcoder.com/challenges/95ae6858-80d7-4e96-b867-81085b150c5a?tab=winners
While other tasks work good:
- https://www.topcoder.com/challenges/d0bcefed-e298-4315-9e88-9c8b8b96c65e?tab=winners
- https://www.topcoder.com/challenges/1d672cbf-d18f-4969-91cd-a14641b6efcc?tab=winners

This doesn't look to me like UI issue and rather backend one.

**Comment by acshields** [comment]
@rootelement - can you review maxceem's notes above and also take a look? Seems like this may be an issue driven by the backend.

**Comment by drasticdpk** [comment]
Still having same issue. 
![image](https://user-images.githubusercontent.com/25297859/105167023-49418f80-5b40-11eb-83e9-376701f7ec3b.png)
will come up with finding if found. 
![image](https://user-images.githubusercontent.com/25297859/105167124-69714e80-5b40-11eb-8eb8-e09988ce5029.png)

@maxceem  are you access this challenge, since this is assigned to you. 
https://www.topcoder.com/challenges/849e4d4f-0c3f-44a9-9ac2-318db739829c?tab=registrants

**Comment by vikasrohit** [comment]
I think we have to live with this bug until we get rid of legacy as per discussions with @rootelement last week, @rootelement please correct me if I am wrong here.

**Comment by rootelement** [comment]
Correct. This is a legacy issue that exists because of the v4 api. There's nothing we can fix or change here.

**Comment by vikasrohit** [comment]
@acshields moving it to Backlog just ot keep track on it.

**Comment by acshields** [comment]
@vikasrohit - noted
